src/Api/NoPersistent.hs

```diff
@@ -0,0 +1,52 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeOperators         #-}
+{-# LANGUAGE ViewPatterns          #-}
+
+{-# OPTIONS_GHC #-}
+
+module Api.NoPersistent
+where
+
+import Control.Monad.IO.Class
+import Control.Concurrent.MVar
+import Control.Lens
+import Data.String (fromString)
+import Network.Wai.Handler.Warp (runSettings, setHost, setPort, defaultSettings)
+import Servant
+import Servant.HTML.Blaze
+import System.IO.Unsafe (unsafePerformIO)
+
+import Thentos.Frontend.Types
```

Comment:
I am going to set up docker based testing on travis to avoid the recompilation of thentos and cabal packages.